web-components/src/components/combobox/ComboBox.ts

```diff
@@ -774,15 +775,15 @@ export namespace ComboBox {
         <button
           type="button"
           class="md-combobox-button clear"
-          aria-label=${this.label}
+          aria-label="Clear button"
```

Comment:
why hard coded ariaLabel? it should be passed from application. so that it can be localized